public/electron-starter.js

```diff
@@ -34,7 +34,7 @@ function createWindow() {
   }));
 
   // Open the DevTools.
-  mainWindow.webContents.openDevTools({ mode: 'detach' });
+  mainWindow.webContents.openDevTools(); //{ mode: 'detach' });
```

Comment:
I just found switching between windows slowed my testing down. But actually we can remove it, there's already a shortcut in our app for dev tools `Cmd+Option+i`